lms/djangoapps/teams/static/teams/js/teams_features.js

```diff
@@ -61,19 +61,43 @@
     };
 
     function buttonAddMembers(staff, url){
-        var button = $("<button class='action action-primary'>Add Members</button>");
+        var button = $("<button class='btn btn-secondary'>Add Members</button>");
         var input = $("<input type='file' name='fileUpload' style='display: none;'accept='text/csv'/>")
         if(!$(".page-header-secondary").children()[0] && staff){
             $(".page-header-secondary").append(input);
             $(".page-header-secondary").append(button);
 
             input.fileupload({
                 url: url,
-                done:function(){
+                done:function(e, data){
                     $(".page-header-secondary").empty();
                     buttonAddMembers(staff, url);
```

Comment:
Sorry don't understand the logic behind calling the method buttonAddMembers inside itself

Reply:
in the previous version was necessary to reload the button because it only works once time, but now This line is not necessary and was removed in last commit